_drafts/bash_completion.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+author: chris
+title: Bash Directory Completion
```

Comment:
Maybe title it something like: Save Time Through Bash Directory Autocompletion?

Reply:
Done. "Save time through bash programmable autocompletion"